src/utils/parseMacros.ts

```diff
@@ -8,6 +8,8 @@ interface Macro {
   endLineNumber: number | null
   declaration: string
   termination: string
+  declarationTrimmedStatement: string
+  terminationTrimmedStatement: string
```

Comment:
Can we not just replace `declaration` and `termination` with these? I don't think we need the non-trimmed statements anywhere in particular.

Reply:
Renaming is good